src/openrct2-ui/interface/Theme.cpp

```diff
@@ -183,6 +183,7 @@ static constexpr const WindowThemeDesc WindowThemeDescriptors[] =
     { THEME_WC(WC_SERVER_LIST),                    STR_SERVER_LIST,                                  COLOURS_2(COLOUR_LIGHT_BLUE,               COLOUR_LIGHT_BLUE                                                                   ) },
     { THEME_WC(WC_CHAT),                           STR_CHAT,                                         COLOURS_1(TRANSLUCENT(COLOUR_GREY)                                                                                             ) },
     { THEME_WC(WC_CONSOLE),                        STR_CONSOLE,                                      COLOURS_2(TRANSLUCENT(COLOUR_LIGHT_BLUE),  COLOUR_WHITE                                                                                       ) },
+    { THEME_WC(WC_FIREWORKS),                      STR_FIREWORKS_EDITOR,                             COLOURS_3(COLOUR_MOSS_GREEN,             COLOUR_MOSS_GREEN,             COLOUR_MOSS_GREEN                                ) },
```

Comment:
It would probably be better to make this window in the plugin framework and just expose what you need to make that happen. Thats what we have done with the title sequence editor which this is similar to.

Reply:
I think we need to wait for the full prototype of fireworks before discussing how much should be part of the base game.